components/avatar/avatar_icon.story.vue

```diff
@@ -4,19 +4,16 @@
     :size="size"
     :color="color"
   >
-    <component
-      :is="defaultSlot"
-    />
+    <dt-icon :name="defaultSlot" />
```

Comment:
Need a conditional in here somewhere to use updated `size` of `<dt-icon>`. Something like..

```
AVATAR SIZE
sm → icon size 200;
md → icon size 400;
lg → icon size 500;
```
<img width="40" alt="image" src="https://user-images.githubusercontent.com/1165933/202036705-fcabee63-c04b-4896-b1bb-9daf59c135ab.png">